### DISCUSSION TITLE
install cask issue, source SSL has expired

### DISCUSSION THREAD

**Discussion by jackietw** [Homebrew/discussions#5415] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.2
ORIGIN: https://github.com/Homebrew/brew
HEAD: 9fdb0d53990d3e62e25325a6f89dbb769e84a530
Last commit: 2 days ago
Core tap JSON: 29 May 08:55 UTC
Core cask tap JSON: 29 May 08:55 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.3.1 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.3.1/bin/ruby
CPU: octa-core 64-bit kabylake
Clang: 15.0.0 build 1500
Git: 2.45.1 => /usr/local/bin/git
Curl: 8.6.0 => /usr/bin/curl
macOS: 14.5-x86_64
CLT: 15.3.0.0.1.1708646388
Xcode: N/A
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: You have the following deprecated, official taps tapped:
  Homebrew/homebrew-cask-fonts
Untap them with `brew untap`.
```

### Description of issue

Warning: Formula homebrew/cask-fonts/font-ocr was renamed to homebrew/cask/font-ocr.
==> Downloading https://osdn.dl.osdn.jp/tsukurimashou/56948/ocr-0.2.zip
curl: (60) SSL certificate problem: certificate has expired
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.

Error: Download failed on Cask 'font-ocr' with message: Download failed: https://osdn.dl.osdn.jp/tsukurimashou/56948/ocr-0.2.zip

### Relevant casks

font-ocr

**Comment by p-linnane** [comment]
There's not much we can do if an upstream certificate expired. We have no control over this.
